src/ctapipe/instrument/optics.py

```diff
@@ -263,3 +270,176 @@ def __repr__(self):
 
     def __str__(self):
         return self.name
+
+
+class PSFModel:
+    def __init__(self, **kwargs):
+        """
+        Base component to describe image distortion due to the optics of the different cameras.
+        """
+
+    @classmethod
+    def from_name(cls, name, **kwargs):
```

Comment:
could be confusing here... In the instrument model, we use `from_name` to load a named model from a file, not to get a sub-class...  this is a general API conflict with the `Component` classes, however, which is a larger issue, but if this is in instrument, it should at least be consistent.